tests/phpunit/tests/functions/_wpCheckExistingFileNames.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @group functions.php
+ * @covers ::wp_is_numeric_array
```

Comment:
```suggestion
 * @covers ::_wp_check_existing_file_names
```
- Corrected `@covers` annotation.